### PR TITLE
ISOC-3776 Installation webhook handler

### PR DESCRIPTION
**Pull request by Harminder84**
**What's in this PR?**
Implemented `installation` webhook handler to check whether security permissions has been accepted or not. If yes, then
 - update`isSecurityPermissionsAccepted` column in `Subscriptions` table to true
 -  Submit request to Jira to link security workspace

**Why**
We are introducing new permissions to access `dependabot_alerts` and `secret_scanning`. Once these permissions are made available in the GH app, the GH Admin will need to accept them. We will use this data to track customers whose permission requests are still pending. Backfill for security VULNS will also be triggered on permission acceptance 

**Added feature flags**
ENABLE_GITHUB_SECURITY_IN_JIRA

**Affected issues**  
[ISOC-3776]

**How has this been tested?**  
Unit test
Local

[ISOC-3776]: https://softwareteams.atlassian.net/browse/ISOC-3776

### CODE REVIEW

File: src/github/installation.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+	}
+};
+
+const hasSecurityPermissions = (permissions: InstallationEvent["installation"]["permissions"]) => {
+	return SECURITY_PERMISSIONS.every(securityPermission => securityPermission in permissions);
+};
+
+const setSecurityPermissionAccepted = async (subscription: Subscription, logger: Logger) => {
+	try {
+		if (subscription) {
```
Comment (gxueatlassian) [inline src/github/installation.ts:95]: no need for `if`?

File: src/github/installation.ts [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+			}
+		},
+		gitHubAppConfig: {
+			gitHubAppId
+		}
+	} = context;
+
+	const jiraHost = jiraClient.baseURL;
+
+	const logger = context.log.child({
```
Comment (gxueatlassian) [inline src/github/installation.ts:37]: I don't think we need those
```
const logger = context.log.child({
		gitHubInstallationId,
		jiraHost
	});
```
Should be already set.

**Review by gxueatlassian — CHANGES_REQUESTED** [review]
Looks good ✅  to me, let me know once db scripts are ran for all env, then I can unblock this one.

**Comment by Harminder84** [comment]
@gxueatlassian, I ran the DB scripts for all three environments.

**Review by gxueatlassian — APPROVED** [review]

**Pull request merged**